lib/utils/zero-config.js

```diff
@@ -1,40 +1,38 @@
 function getEnvFromOptionsAndMode(mode, optionsObject) {
-    const optionsDevFlag = optionsObject.dev;
-    const optionsProdFlag = optionsObject.prod;
+    const { dev, prod } = optionsObject;
 
     if (process.env.NODE_ENV && process.env.NODE_ENV === 'production' || process.env.NODE_ENV === 'development') {
```

Comment:
```js
if (process.env.NODE_ENV && ['production', 'development'].includes(process.env.NODE_ENV)) {
```
Maybe :thinking:

Reply:
I assume, includes will be slower here. (eventhough it is very negligible but still)

Reply:
create a new array just for a comparison? Not needed I think